.gitignore

```diff
@@ -2,3 +2,5 @@
 *.pyc
 data
 node_modules
+manadex
```

Comment:
Oh, right, just like the instructions say!  I do it differently, I'll show you sometime.  Of course, only works on *nix.